open-metadata-implementation/common-services/ffdc-services/src/main/java/org/odpi/openmetadata/commonservices/ffdc/rest/Parametrization.java

```diff
@@ -1,27 +1,32 @@
 /* SPDX-License-Identifier: Apache-2.0 */
 /* Copyright Contributors to the ODPi Egeria project. */
-package org.odpi.openmetadata.accessservices.subjectarea.client;
+package org.odpi.openmetadata.commonservices.ffdc.rest;
 
-import org.odpi.openmetadata.accessservices.subjectarea.responses.SubjectAreaOMASAPIResponse;
 import org.springframework.core.ParameterizedTypeReference;
 import org.springframework.core.ResolvableType;
 
 /**
- * @param <T> The type of the parameterized class whose objects will be returned in responses.
+ * @param <Result> The type of the parameterized class whose objects will be returned in responses.
  * The purpose of this interface is to enable capturing and passing a generic type. */
-public interface Parametrization<T> {
+public interface Parametrization<Result> {
```

Comment:
Shouldn't the spelling of this be Parameterization

Reply:
I this case it generic parameter is mark of result

Reply:
the interface name is Parametrization, but this is not a proper word , the closest word is Parameterization. I wondered if that was what you intended.

Reply:
Ah okey, sorry for my English...

Reply:
"In mathematics, and more specifically in geometry, parametrization (or parameterization; also parameterisation, parametrisation)". I read it in https://en.wikipedia.org/wiki/Parametrization

Reply:
> Parametrization, also spelled parameterization, parametrisation or parameterisation, is the process of defining or choosing parameters. I read it in https://en.wikipedia.org/wiki/Parametrization

Nice evidence based response .